app/assets/javascripts/oxalis/model/binary/bucket_picker_strategies/orthogonal_bucket_picker.js

```diff
@@ -112,29 +112,32 @@ function addNecessaryBucketsToPriorityQueueOrthogonal(
 
     // Build up priority queue
     wSliceOffsets.forEach(wSliceOffset => {
-      for (
-        let y = scaledTopLeftVector[v] - extraBucket;
-        y <= scaledBottomRightVector[v] + extraBucket;
-        y++
-      ) {
-        for (
-          let x = scaledTopLeftVector[u] - extraBucket;
-          x <= scaledBottomRightVector[u] + extraBucket;
-          x++
-        ) {
+      const extraYBucketStart = scaledTopLeftVector[v] - extraBucket;
+      const extraYBucketEnd = scaledBottomRightVector[v] + extraBucket;
+      const extraXBucketStart = scaledTopLeftVector[u] - extraBucket;
+      const extraXBucketEnd = scaledBottomRightVector[u] + extraBucket;
+
+      for (let y = extraYBucketStart; y <= extraYBucketEnd; y++) {
+        for (let x = extraXBucketStart; x <= extraXBucketEnd; x++) {
           const bucketAddress = ((topLeftBucket.slice(): any): Vector4);
           bucketAddress[u] = x;
           bucketAddress[v] = y;
           bucketAddress[w] += wSliceOffset;
 
           const bucket = binary.cube.getOrCreateBucket(bucketAddress);
+          const isExtraBucket =
```

Comment:
I can't think of a more elegant way to express the logic for the "extra" bucket (better naming needed, too ^^).

Reply:
What do you think about simply using the [Chebyshev distance](https://lyfat.wordpress.com/2012/05/22/euclidean-vs-chebyshev-vs-manhattan-distance/) instead of the manhattan distance to assign priorities for the priority queue? I feel like this would reduce complexity and avoid the need to take special care of the "extra" buckets :)

Reply:
That would indeed solve the problem. However, we would change the loading behavior (from the user's point of view) quite a bit :thinking: Not sure whether that's ok..

Reply:
I can't remember what the conclusion of our discussion about this yesterday was :see_no_evil:

Reply:
I think we said that the priority here doesn't really matter as the ping strategy takes precedence, anyway. However, it's good to leave this edge case in the code because the code might make the ping strategy obsolete at some point (and then it's important). I wouldn't switch to chebyshev without having the approval from the lab..